scenario/askhubadvisor.go

```diff
@@ -164,12 +164,11 @@ func (settings *AskHubAdvisorSettings) UnmarshalJSON(bytes []byte) error {
 	}
 	switch settings.QuerySource {
 	case QueryFromFile:
-
 		if settings.FileName == "" {
 			return errors.New("no file name")
 		}
 		if runtime.GOOS == "js" {
-			return errors.Errorf("can not read file with GOOS=%s", runtime.GOOS)
+			return nil
 		}
 		file, err := os.Open(settings.FileName)
```

Comment:
Shouldn't this be moved to execute?

Reply:
`os.Open` I mean

Reply:
If it's in Execute it will be executed for each user though? I guess there may be some way around that but it doesn't really make a difference. Having it in Umarshal is fine as long as it doesn't run in js.